assets/components/productPage/productPageHero/productPageHero.jsx

```diff
@@ -12,21 +12,22 @@ import { classNameWithModifiers } from 'helpers/utilities';
 
 type PropTypes = {|
   overheading: string,
+  type: 'grey' | 'feature',
   heading: string,
-  headline: string | null,
-  cta: Node | null,
-  children: Node | null,
+  headline?: string | null,
+  cta?: Node | null,
+  children?: Node | null,
```

Comment:
There's an `Option` type available for this now at `helpers/types/option` if you prefer.